.hound.yml

```diff
@@ -0,0 +1,2 @@
+ruby:
+  config_file: .rubocop.yml
```

Comment:
Added HoundCI to the project. Super simple CI tool for enforcing codestyles on a pull request basis. A bot will commet on your pull requests to follow the proper coding style set out by this file (or use the default linters). It's free for open source and sufia/hydra is also using this.